publish_trips.py

```diff
@@ -47,12 +67,19 @@ def getLogger(name, level=logging.INFO):
 
 
 def get_max_socrata_date(resource_id):
-    url = "https://data.austintexas.gov/resource/{}.json?$query=SELECT checkout_date as date ORDER BY checkout_date DESC LIMIT 1".format(
-        resource_id
-    )
-    res = requests.get(url)
+    url = f"https://data.austintexas.gov/resource/{resource_id}.json"
+    params = {
+        "$query": "SELECT checkout_date as date where checkout_date is not null ORDER BY checkout_date DESC LIMIT 1"
```

Comment:
the key change here is `not null`. the crufty records with a `null` checkout_date were breaking the script because they are sorted before any record with a checkout date.